src/geff/interops/__init__.py

```diff
@@ -0,0 +1,33 @@
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from .ctc import ctc_tiffs_to_zarr, from_ctc_to_geff
+    from .trackmate_xml import from_trackmate_xml_to_geff
```

Comment:
```suggestion
from geff.interops.trackmate_xml import from_trackmate_xml_to_geff

if TYPE_CHECKING:
    from .ctc import ctc_tiffs_to_zarr, from_ctc_to_geff
```

no longer anything in trackmate_xml that needs an extra